tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -91,18 +91,28 @@ Status TrtCandidateSelector::IsTensorRTCandidate(const tensorflow::Node* node) {
   // LINT.IfChange
   static const std::set<string> candidate_ops = {
```

Comment:
we can now make use of UnaryOperationMap to construct candidate_ops without repeating the list of names.

Reply:
I think the plan is to remove this list entirely when all ops have validators, that way we only need the  one list in RegisterValidatableOpConverters(). I think its okay to have them duplicated for now? There are only about 5 or so more ops that need validators so it won't be long. What do you think?

Reply:
Sounds good. Thanks for the explanation.